src/decorators/db.ts

```diff
@@ -0,0 +1,32 @@
+import Elysia from "elysia";
+import { DbAction } from "../types/database";
```

Comment:
```suggestion
import { DbAction } from "@types/database";
```

Optional: to have a nice clean import you could use `module resolution` in TS: https://www.typescriptlang.org/tsconfig#moduleResolution